README.md

```diff
@@ -1,13 +1,187 @@
-OpenPipeline
-================
+# OpenPipeline
 
-<!-- README.md is generated by running 'quarto render README.qmd' -->
+OpenPipeline are extensible single cell analysis pipelines for reproducible and large-scale single cell processing using Viash and Nextflow.
 
-Extensible single cell analysis pipelines for reproducible and
-large-scale single cell processing using Viash and Nextflow.
+[![GitHub](https://img.shields.io/badge/GitHub-openpipelines--bio%2Fopenpipeline-blue.svg)](https://github.com/openpipelines-bio/openpipeline)
+[![GitHub
+License](https://img.shields.io/github/license/viash-hub/demultiplex.svg)](https://github.com/viash-hub/demultiplex/blob/main/LICENSE)
+[![GitHub
+Issues](https://img.shields.io/github/issues/openpipelines-bio/openpipeline.svg)](https://github.com/viash-hub/demultiplex/issues)
+[![Viash
+version](https://img.shields.io/badge/Viash-v0.9.4-blue)](https://viash.io)
 
-The provided pipelines are built using the [Viash
-framework](http://www.viash.io) on top of the nextflow workflow system.
-For more information on Nextflow please visit the [Nextflow github
-page](https://github.com/nextflow-io/nextflow) and the [Nextflow read
-the docs page](https://www.nextflow.io/docs/latest/index.html).
+## Documentation
+
+Please find more in-depth documentation on [the website](https://openpipelines.bio/).
+
+## Functionality Overview
+
+Openpipelines execute a list of predefined tasks. These descrete steps are also provided as standalone components that can be executed individually, with a standardized interface. This is especially useful when a particular step wraps a tool that you do not necessarily always need to execute in a workflow context.
+
+In terms of workflows, the following functionality is provided:
+
+* Demultiplexing: conversion of raw sequencing data to FASTQ objects.
+* Ingestion: Read mapping and generating a count matrix.
+* Single sample processing: cell filtering and doublet detection.
+* Multisample processing: Count transformation, normalization, QC metric calulations.
+* Integration: Clustering, integration and batch correction using single and multimodal methods.
+* Downstream analysis workflows
```

Comment:
Could you add a mermaid diagram with links to respective documentation pages here?

Reply:
Done